docs/userguide/modules/exposing-binding-data/pages/supported-operators.adoc

```diff
@@ -0,0 +1,26 @@
+= Known bindable operator-backed services
+
+The {servicebinding-title} can automatically detect and bind to services
+created by a limited selection of operators. These operators do not support
+binding directly. Instead, the {servicebinding-title} is able to detect and
+configure the operator's CRDs so that they become bindable.
+
+The long-term intention is to contribute upstream support for service binding
+and remove the operators that gain native support for service bindings. 
+
+The operators that currently fall in this category are:
+
+* link:https://operatorhub.io/operator/redis-operator[OpsTree Redis]:
```

Comment:
not sure if `link:` prefix is really mandatory, I know that asciidoc links are simply `https://operatorhub.io/operator/redis-operator[OpsTree Redis]`

Reply:
Perhaps is nicer if we convert these bullet points into a table, something like:

| Operator | Supported GVKs |
| -- | -- |
| OpsTree Redis | Redis.redis.redis.opstreelabs.in/v1beta1 |

Reply:
the section was removed, eventually